### PR TITLE
Updating CI job to run integration tests in V1

### PR DESCRIPTION
**Pull request by EricCornelson**

### CODE REVIEW

File: .vsts/job.yml [hunk 1]
```diff
@@ -61,14 +61,14 @@ jobs:
...
-    - task: Npm@1
-      condition: false
-      displayName: npm frameworkTest
-      inputs:
-        command: custom
-        verbose: false
-        customCommand: 'run frameworkTest'
+  #- ${{ if eq(parameters.name, 'Windows') }}:
+  - task: Npm@1
+    condition: false
```
Comment (digeff) [inline .vsts/job.yml]: Shouldn't be remove this to have the tests run?

Comment (EricCornelson) [reply .vsts/job.yml]: Yep! :)

File: .vsts/job.yml [hunk 2]
```diff
@@ -102,10 +95,25 @@ jobs:
...
     - task: PublishTestResults@2
+      condition: always()
       inputs:
         testResultsFormat: 'JUnit' # Options: JUnit, NUnit, VSTest, xUnit, cTest
         testResultsFiles: '**/test-*.xml'
+        publishRunAttachments: true
 
+  - ${{ if eq(parameters.name, 'Windows') }}:
+    - task: PublishPipelineArtifact@0
+      condition: always()
```
Comment (roblourens) [inline .vsts/job.yml]: You don't need condition. You could use that instead of the if block for windows if you want

Comment (EricCornelson) [reply .vsts/job.yml]: Done

**Pull request merged**